source/nvda.pyw

```diff
@@ -63,13 +65,18 @@ import locale
 import gettext
 
 try:
-	gettext.translation(
+	trans = gettext.translation(
 		'nvda',
 		localedir=os.path.join(globalVars.appDir, 'locale'),
 		languages=[locale.getdefaultlocale()[0]]
-	).install(True)
+	)
+	trans.install(True)
+	# Install our pgettext function.
+	builtins.pgettext = makePgettext(trans)
 except:
 	gettext.install('nvda')
+	# Install our pgettext function.
+	builtins.pgettext = lambda ctx, msg: msg

```

Comment:
This should be consistent with the other instance in the unit tests
It might be worth turning this into a named function e.g. `def pgettextNoTranslation`

```suggestion
	builtins.pgettext = lambda context, message: message
```

Reply:
I have added your suggestion in 61af5f252be7acb40e51cb7aef5f0992a90b3e79. I have kept this function anonymous. Defining a named function here would make the code more heavy IMO. And I did not want either to add an extra import for this function (its always not very clear when things need to be imported in nvda.ppw without risking to import things too early).

Reply:
Also, should I add the same `pgettext` function in `nvda_slave.pyw`? I have not found any case where `pgettext` missing causes a problem there; but maybe its better to handle `nvda.pyw` and `nvda_slave.pyw` the same way.

What do you think?

Reply:
Probably best to add it to `nvda_slave` just in case

Reply:
Done in 214c4e4efb767fd6110620c90d3c02000627fd9d.